<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VersionDialog</class>
 <widget class="QDialog" name="VersionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frameVersionSetting">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>580</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_SoftwareVersion">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>213</y>
     <width>495</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;
font: 24px</string>
   </property>
   <property name="text">
    <string>1.0-20120213</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>143</y>
     <width>205</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26px</string>
   </property>
   <property name="text">
    <string>硬件版本</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>213</y>
     <width>205</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26px</string>
   </property>
   <property name="text">
    <string>软件版本</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>73</y>
     <width>205</width>
     <height>70</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 26px</string>
   </property>
   <property name="text">
    <string>设备型号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_HardwareVersion">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>143</y>
     <width>495</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;
font: 24px</string>
   </property>
   <property name="text">
    <string>1.0-20120101</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_ModalType">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>73</y>
     <width>495</width>
     <height>70</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 0px;
background-color: transparent;
font: 24px</string>
   </property>
   <property name="text">
    <string>BASIC-TERMINAL</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>10</y>
     <width>510</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>版本</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
